app/controllers/github_events_controller.rb

```diff
@@ -28,14 +28,14 @@ def process_pull_request_comment
     comment = Comment.new(event.comment_hash)
     owner = pull_request.github_user
 
-    if comment.commenter.reviews?(pull_request)
-      review = comment.commenter.reviews.find_by(pull_request: pull_request)
-      review.reject if comment.rejected?
-      review.approve if comment.approved?
-    elsif comment.commenter == owner
+    if comment.commenter == owner
```

Comment:
Not here, these are user objects. But do you mean in the algorithm that parses the comment text for mentions?